frontend/pages/policies/PolicyPage/components/PolicyForm/PolicyForm.tsx

```diff
@@ -159,6 +159,10 @@ const PolicyForm = ({
     });
 
     if (valid) {
+      if (!policyIdForEdit) {
```

Comment:
Is this needed or it is picked up by line 166 below? I think this is essentially the same thing as `!isEditMode`.

Reply:
I just pulled in your changes for this page! Thanks for catching this! <3